### PR TITLE
Update Simplified Chinese translations

### PR DESCRIPTION
**Pull request by ghost**
Looks like I'm a bit late on this... hope there's gonna be a v2.1.3 soon :rofl: 
Feel free to remind me if I missed anything since v2.1.0 release.

### CODE REVIEW

File: config/locales/zh-CN.yml [hunk 1]
```diff
@@ -263,12 +263,18 @@ zh-CN:
       unresolved: 未处理
       view: 查看
     settings:
+      activity_api_enabled:
+        desc_html: 在本站发布的嘟文数、活跃用户数以及一周内新用户数
+        title: 公开用户活跃度的统计数据
```
Comment (SerCom-KC) [inline config/locales/zh-CN.yml]: Review note: `user activity` → `用户活跃度`

File: config/locales/zh-CN.yml [hunk 2]
```diff
@@ -263,12 +263,18 @@ zh-CN:
...
       bootstrap_timeline_accounts:
         desc_html: 用半角逗号分隔多个用户名。只能添加来自本站且未开启保护的帐户。如果留空，则默认关注本站所有的管理员。
         title: 新用户默认关注
       contact_information:
         email: 用于联系的公开电子邮件地址
         username: 用于联系的公开用户名
+      peers_api_enabled:
+        desc_html: 截至目前本实例在网络中已发现的域名
```
Comment (SerCom-KC) [inline config/locales/zh-CN.yml]: Review note: `fediverse` → `网络`，`encountered` → `发现`

**Comment by ykzts** [comment]
cc @Artoria2e5 @KenOokamiHoro @leopku @miuchan @vidarlee

**Review by Artoria2e5 — APPROVED** [review]
good notes btw

**Review by ykzts — APPROVED** [review]

**Pull request merged**